bin/install.sh

```diff
@@ -179,7 +183,12 @@ hasproj() {
 
 export JULIA_PROJECT="@."
 
-cd "${CIRRUS_WORKING_DIR}"
+if [ -z "${JULIA_PROJECT_SUBDIR}" ]; then
+    cd "${CIRRUS_WORKING_DIR}"
+else
+    cd "${CIRRUS_WORKING_DIR}/${JULIA_PROJECT_SUBDIR}"
+fi
```

Comment:
You should be able to unconditionally use `JULIA_PROJECT_SUBDIR` since `cd` will still work correctly if that's empty:
```suggestion
cd "${CIRRUS_WORKING_DIR}/${JULIA_PROJECT_SUBDIR}"
```